packages/tools/devtools/devtools-view/src/TelemetryUtils.ts

```diff
@@ -44,6 +45,8 @@ export class ConsoleVerboseLogger implements ITelemetryBaseLogger {
 	public send(event: ITelemetryBaseEvent): void {
 		// Deliberately using console.debug() instead of console.log() so the events are only shown when the console's
 		// verobsity level is set to "Verbose".
+		event.PKG_VERSION = pkgVersion;
```

Comment:
give this a better name, many logs already have versions like driverVersion, or runtimeVersion, maybe devtoolsVersion

Reply:
Ah yes, I missed commenting on that piece. `devtoolsVersion` seems like a good option. FWIW @anthony-murphy , this is for the devtools usage telemetry only, it won't show up in the current logs. And the plan is to put them in a different table(s). Not sure there'll be a lot of use cases where we'll want to query both sets at the same time.